src/layouts/Layout.astro

```diff
@@ -259,7 +259,7 @@ const { title, navless } = Astro.props;
     button {
       display: inline-block;
       padding: 0.4rem 0.8rem;
-      color: var(--color-bg);
+      color: black;
```

Comment:
Are we really using pure black? Surprised if so.

Reply:
Me too — the text colour in yesterday's badge brief was `#000000`. Shall I change it to `--color-bg`?